bin/pygrb/pycbc_grb_trig_cluster

```diff
@@ -168,105 +168,143 @@ outfile = os.path.join(
     ),
 )
 
-# -- generate clustering bins -------------------
+# this list contains the indexing of clusters from all slides
+all_clusters = []
 
-nbins = int((end - start) // win + 1)
-bins = [[] for i in range(nbins)]
-loudsnr = numpy.zeros(nbins)
-loudtime = numpy.zeros(nbins)
-clusters = []
+# load necessary information from all triggers
 
-# -- cluster ------------------------------------
```

Comment:
Do we still need this comment ?

Reply:
Yes, because this where we cluster.  I will add a comment to flag where the output is written.

Reply:
Sorry, I commented on lines you removed. Now I see the comment in line 189 of your new code